R/model_from_api.R

```diff
@@ -97,3 +97,48 @@ model_from_api <- function(url,
     return(mod)
   }
 }
+
+#' Install default literature model
+#'
+#' A very lightweight wrapper for `model_from_api` that installs previously
+#' published models packaged within PKPDsim.
+#'
+#' @param model Name of model, e.g., "pk_busulfan_mccune". See
+#' [available_default_literature_models()]
+#' @param ... arguments passed onto `model_from_api`. For fine-grain control, it
+#'   is better to install models directly from [model_from_api()] or
+#'   [new_ode_model()].
+#' @examples \dontrun{
+#' install_default_literature_model("pk_busulfan_mccune")
+#' }
+#' @export
+
+install_default_literature_model <- function(model, ...) {
```

Comment:
I would probably separate into its own file? I see that it is sort of a helper function for model_from_api, but it's an exported one and kind of separate functionality?